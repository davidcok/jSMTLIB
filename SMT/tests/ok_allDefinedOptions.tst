; Tests the defaults for all defined options.  Some of these may be unsupported.
(get-option :print-success)
(get-option :random-seed)
(get-option :expand-definitions)
(get-option :interactive-mode)
(get-option :produce-proofs)
(get-option :produce-unsat-cores)
(get-option :produce-models)
(get-option :produce-assignments)
(get-option :regular-output-channel)
(get-option :diagnostic-output-channel)
(get-option :verbosity)

