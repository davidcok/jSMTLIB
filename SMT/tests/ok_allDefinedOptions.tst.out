true
0
false
false
false
false
false
false
"stdout"
"stderr"
0
